nucypher/blockchain/eth/utils.py

```diff
@@ -14,6 +14,7 @@
 You should have received a copy of the GNU Affero General Public License
 along with nucypher.  If not, see <https://www.gnu.org/licenses/>.
 """
+from datetime import datetime
```

Comment:
This import is unused because we use everything through maya

Reply:
:heavy_check_mark: